pyproject.toml

```diff
@@ -49,11 +52,12 @@ allow_redefinition = true
 
 # Ignore libraries that do not have type hint nor stubs
 [[tool.mypy.overrides]]
-module = ["ops.*", "pytest_operator.*"]
+module = ["ops.*", "pytest_operator.*", "requests.packages.urllib3.*"]
 ignore_missing_imports = true
+follow_imports = "skip"
 
 [[tool.mypy.overrides]]
-module = ["charms.alertmanager_k8s.v0.*", "charms.grafana_k8s.v0.*"]
+module = ["charms.prometheus_k8s.*"]
```

Comment:
After @rbarry82 publishes the latest prom lib you could probably remove this override.

Reply:
I'll leave for now and look back in to this after beta release.